src-electron/generator/helper-attribute.js

```diff
@@ -172,3 +206,4 @@ exports.global_attribute_default = attributeDefault
 exports.feature_bits = featureBits
 exports.as_underlying_atomic_identifier_for_attribute_id =
   as_underlying_atomic_identifier_for_attribute_id
+exports.count_mandatory_attributes = count_mandatory_attributes
```

Comment:
If this is Matter only then switch this to have matter in the name. count_mandatory_matter_attributes